src/EMPhysicsPolarizedJAEA.cc

```diff
@@ -45,7 +45,7 @@ along with utr.  If not, see <http://www.gnu.org/licenses/>.
 
 // gamma
 #include "G4PhotoElectricEffect.hh"
-#include "G4LivermorePolarizedPhotoElectricModel.hh"
+// #include "G4LivermorePolarizedPhotoElectricModel.hh" // Commented out for compatibility with Geant4 v11, which removed this "obsolete unused class"
```

Comment:
If it’s obsolete and unused, why not remove it?

Reply:
Because I'm not 100% sure how the whole physics list business works and whether this line had an effect in Geant4 v10.
My hope is someone more proficient with custom physics were to use this custom physics list, this comment can help to understand why the line is not included here...